exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,5 +1,8 @@
 // Declare your function first
-
+function divide(num1, num2) {
+  return num1 / num2;
+}
 var result = divide(3, 4);
 
 console.log(result);
+exercises / K - functions - parameters / exercise3.js;
```

Comment:
Code looks good, just be careful not to leave in extra stuff in the code like here on line 8 :smile:

Reply:
Thank you for the review